packages/jbrowse-plugin-apollo/src/FeatureDetailsWidget/ApolloTranscriptDetailsWidget.tsx

```diff
@@ -112,6 +119,45 @@ export const ApolloTranscriptDetailsWidget = observer(
       props,
     ) as React.ElementType<CustomComponentProps>
 
+    const BiotypesComponent = pluginManager.evaluateExtensionPoint(
+      'Apollo-BiotypesComponent',
+      () => null,
+      {
+        key: 'status',
+        feature,
+        session,
+        assembly,
+        TypeChange,
+        DeleteFeatureChange,
+        AddFeatureChange,
+        FeatureAttributeChange,
```

Comment:
Instead of passing these changes, the plugin using this extension point should add the `@apollo-annotation/shared` package (it's published to NPM so anyone can use it) and import the changes from there.

Reply:
Latest package doesn't contain the recent FeatureAttributeChangeDetails changes. Is it possible to publish that package separately ?

Reply:
Yeah, I'll work on getting a release out.

Reply:
You can now use the released v0.3.7